core/src/main/java/ai/timefold/solver/core/impl/score/director/InnerScoreDirector.java

```diff
@@ -228,17 +230,6 @@ default Score_ doAndProcessMove(Move<Solution_> move, boolean assertMoveScoreFro
      */
     SupplyManager getSupplyManager();
 
-    /**
```

Comment:
Should we deprecate it before removing it?

Reply:
Same answer as above. I can afford to remove it without deprecating, because this is internal code. Nobody uses this method, as evidenced by the code compiling.